server/.env

```diff
@@ -1,6 +1,3 @@
-
 JWT_SECRET=SECRET
 NODE_ENV=development
```

Comment:
can we work out why this is in the repo please

Reply:
no idea since it's in all of the .gitignores

Reply:
removed it now